### PR TITLE
Added config to Review and Organization.

### PR DESCRIPTION
**Pull request by MishkaSorochinskiy**

### CODE REVIEW

File: DeratControl.Ifrastructure/EntitiesConfigurations/OrganizationConfiguration.cs [hunk 1]
```diff
@@ -1,10 +1,23 @@
...
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
 
 namespace DeratControl.Infrastructure.EntitiesConfigurations
 {
-    class OrganizationConfiguration
+    class OrganizationConfiguration : IEntityTypeConfiguration<Organization>
     {
+        public void Configure(EntityTypeBuilder<Organization> builder)
+        {
+            builder.HasKey(o => o.Id);
```
Comment (SviatoslavMykhailiv) [inline DeratControl.Ifrastructure/EntitiesConfigurations/OrganizationConfiguration.cs]: think of moving common properties of all the entities into separate base entity configuration class

**Review by SviatoslavMykhailiv — APPROVED** [review]

File: DeratControl.Ifrastructure/EntitiesConfigurations/OrganizationConfiguration.cs (hunk 1 above)
Comment (MishkaSorochinskiy) [reply DeratControl.Ifrastructure/EntitiesConfigurations/OrganizationConfiguration.cs]: ok)

Comment (MishkaSorochinskiy) [reply DeratControl.Ifrastructure/EntitiesConfigurations/OrganizationConfiguration.cs]: Done)

Comment (SviatoslavMykhailiv) [reply DeratControl.Ifrastructure/EntitiesConfigurations/OrganizationConfiguration.cs]: sweet)

**Pull request merged**